src/app/bav/controllers/confirmDetails.js

```diff
@@ -2,19 +2,15 @@ const BaseController = require("hmpo-form-wizard").Controller;
 const { formatSortCode } = require("../utils");
 
 class ConfirmDetailsController extends BaseController {
-  locals(req, res, callback) {
+  locals(req, res) {
     super.locals(req, res, (err, locals) => {
-      if (err) {
-        return callback(err, locals);
-      }
```

Comment:
Out of interest how come we've removed this?

Reply:
Viveak flagged a callback function call in the landing page in one of the comments above, so I removed here too the two files were in line. I got rid of the if statement too but can reinstate? What do you think